cmd/init.go

```diff
@@ -28,7 +27,7 @@ func createInitCommand(initializer tp.Initializer) *cobra.Command {
   notification_info:
     host_name: "Put your machine's host name"
     default_reminder_schedule:
-      - "*/15 * * * *"
+      - "*/30 * * * *"
```

Comment:
Why did the reminder schedule change?
It looks like a different fix than what this PR is for!

Reply:
because current reminder sending notification way too often and so I think it is good to fix that too as well. it's merely reminder schedule and I mentioned it on the PR SUMMARY.

Reply:
@meetrick agree.

Reply:
It's removed from this PR.